routing/road_graph.hpp

```diff
@@ -54,25 +54,41 @@ inline bool AlmostEqualAbs(Junction const & lhs, Junction const & rhs)
 /// The Edge class represents an edge description on a road network graph
 class Edge
 {
+  enum Flags : uint8_t
+  {
+    Real = 1,        // An edge corersponds to some real segment if raised.
+    PartOfReal = 2,  // An edge is a part of some real sement if raised.
+
+    // Three combinations are possible:
+    // Real and PartOfReal are raised -- a real edge.
+    // Real is not rised, PartOfReal is raised -- a fake edge that is a part of a real one.
+    // Both not raised -- a fake edge that is not a part of any real segment.
```

Comment:
Probably it's worth moving this comment above the line `enum Flags : uint8_t`

Reply:
Ok. Done.